SwifterSwift.xcodeproj/project.pbxproj

```diff
@@ -241,11 +241,7 @@
 		07C50D491F5EB04700F46E5A /* UINavigationBarExtensionTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 07C50D161F5EB03200F46E5A /* UINavigationBarExtensionTests.swift */; };
 		07C50D4A1F5EB04700F46E5A /* UINavigationControllerExtensionsTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 07C50D171F5EB03200F46E5A /* UINavigationControllerExtensionsTests.swift */; };
 		07C50D4B1F5EB04700F46E5A /* UINavigationItemExtensionsTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 07C50D181F5EB03200F46E5A /* UINavigationItemExtensionsTests.swift */; };
-		07C50D4C1F5EB04700F46E5A /* UISearchBarExtensionsTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 07C50D191F5EB03200F46E5A /* UISearchBarExtensionsTests.swift */; };
```

Comment:
Please revert all of these changes.

Reply:
Will do, but I feel like this should be PRed separately. There were a bunch of tests wrapped in `#if os(iOS)` added to the tvOS target. They'd never run because there isn't anything to compile for that platform, but because they exist and didn't pass anything, it'd make Xcode think the tests for tvOS weren't entirely successful